cmd/build.go

```diff
@@ -115,9 +115,7 @@ func NewBuildCmd(flags *flags.GlobalFlags) *cobra.Command {
 
 	// TESTING
 	buildCmd.Flags().BoolVar(&cmd.ForceBuild, "force-build", false, "TESTING ONLY")
```

Comment:
nevermind, understood why but can we still keep the flag?

Reply:
I can re-add the flag if you prefer but I did that because the flag was hidden and it would lead to have two ways to do the same thing (force to use buildkit). I tend to follow KISS reason why I removed it.

Reply:
fixed

Reply:
🙇 thanks - it's technically not necessary but I'd rather not break existing scripts